data_and_models/pipelines/ner/config.cfg

```diff
@@ -1,6 +1,4 @@
-# File generated with: spacy init config config.cfg -l en -p ner -o accuracy.
-# Then adapted to resume the training of the NER component of a spaCy pipeline.
-# The base spaCy pipeline is from scispaCy v0.4.0.
+# Initialized with: spacy init config config.cfg -l en -p lemmatizer,ner -o accuracy -G.
```

Comment:
Thanks for this explanation! I launched the command to do the comparison between this `config.cfg` and ours.
It seems that the main changes are:
1. The rule of the lemmatizer (from mode = `rule` to `lookup`)
2. The transformer model (from `roberta-base` to our model)
3. The `training.score_weights` are different

Am I right that those are the only changes from the default `config.cfg` ? For me to understand, could you explain why 1. and 3. were needed ?

Reply:
Yes, they are the only changes. The defaults are used to limit the complexity of the changes introduced by the PR.

(1) The Lemmatizer is needed for our EntityRuler. The `mode = "rule"` requires a `Tagger` (see the [docs](https://spacy.io/api/lemmatizer)). However, we don't train such component. Fortunately, there are dictionaries of lemmas. One could use them with `mode = "lookup"` (using data from `spacy-lookups-data`).

(3) The default `config.cfg` includes the `training` of the `lemmatizer` component. Therefore, the accuracy for lemmatization (i.e. `lemma_acc`) is taken into account to select the best pipeline during training. We do `not` train the lemmatizer component. So, we don't need `lemma_acc` in `[training.score_weights]`.

Reply:
Thanks for the reply. It clarifies a lot!